Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+
+#if !defined(Q_MOC_RUN)
+#include <Editor/Plugins/Cloth/ClothJointWidget.h>
+#include <Editor/Plugins/HitDetection/HitDetectionJointWidget.h>
+#include <UI/PropertyEditor/ReflectedPropertyEditor.hxx>
+#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/DockWidgetPlugin.h>
+#include <Editor/Plugins/SkeletonOutliner/SkeletonOutlinerBus.h>
+#include <AtomToolsFramework/Inspector/InspectorWidget.h>
+#include <Editor/Plugins/Ragdoll/RagdollNodeWidget.h>
+#include <QPushButton>
+#endif
+namespace EMStudio {
+    class ActorInfo;
+    class NodeInfo;
+}
+namespace EMotionFX
+{
+    class SkeletonOutlinerPlugin;
+    class AttributesWindowRequests
+        : public AZ::EBusTraits
+    {
+    public:
+        virtual bool IsLocked() const { return false; }
+        virtual QModelIndex GetModelIndex() const { return {}; }
+    };
+
+    using AttributesWindowRequestBus = AZ::EBus<AttributesWindowRequests>;
+
+    class JointPropertyWidget
```

Comment:
same here, add a few comments explain what the class does.

Reply:
"A Widget in the Inspector Pane displaying Attributes of selected Nodes in a Skeleton"  good enough?

Reply:
That sounds good to me :+1: